homeassistant/components/stream/__init__.py

```diff
@@ -338,7 +338,6 @@ def _run_worker(self) -> None:
                 )
             except StreamWorkerError as err:
                 self._logger.error("Error from stream worker: %s", str(err))
-                self._available = False
```

Comment:
Is it ok to update this from the thread or should we use `hass.add_job` here as well?

Reply:
It's not ok to update this from the worker thread. But I'm removing this not adding it?

Reply:
Whoops sorry missed that. No wonder it didn't make sense to me why it was there.